### PR TITLE
Update deprecated API

### PR DESCRIPTION
**Pull request by maukoquiroga**
Depended upon by openfisca/openfisca-core#1015

* Technical improvement.
* Details:
  - Update code using deprecated parts of OpenFisca-Core Public API

- - - -

These changes:

- Adapt to the current OpenFisca-Core API

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+### 3.13.2 - [#121](https://github.com/openfisca/country-template/pull/121)
+
+* Minor change
+* Details:
+  - Delete macOS related comment in bootstrap script
```
Comment (HAEKADI) [inline CHANGELOG.md:7]: There was a mix-up with the previous minor version I think 😅

**Review by HAEKADI — APPROVED** [review]
I ran the tests and it looks good to me 🚀

File: CHANGELOG.md (hunk 1 above)
Comment (MattiSG) [reply CHANGELOG.md:7]: This description is not correct 😉

**Review by MattiSG — CHANGES_REQUESTED** [review]
Since this update is made necessary to reflect an OpenFisca-Core version change, the OpenFisca-Core dependency version should be bumped 🙂

**Comment by MattiSG** [comment]
Even though the tests pass, I don't understand how this changeset can be compatible with the current version of Core since only in https://github.com/openfisca/openfisca-core/pull/1015/files#diff-f81fa9f88e2aa6671067dccf0760afe4aca86ef0e6f4a03ee9da39a061077043 is the `model_api` importing `ParameterScaleBracket` 🤔

**Comment by maukoquiroga** [comment]
Superseded by #127

**Pull request closed without merging**